_sources/Strings/Exercises.rst

```diff
@@ -75,17 +75,28 @@ Exercises
 
            .. actex:: ex_8_3
 
-              Assign to a variable in your program a triple-quoted string that contains
-              your favorite paragraph of text - perhaps a poem, a speech, instructions
-              to bake a cake, some inspirational verses, etc.
+                Assign to a variable in your program a triple-quoted string that contains
```

Comment:
Thanks for updating the indentation.  It makes life better for everyone when it is consistent!

Reply:
Thank you! No problem :)